LICENSE

```diff
@@ -1,5 +1,6 @@
 MIT License
 
+Copyright (c) 2024 Saurabh Srivastava
```

Comment:
You don't need to add individual contributors to the license file, they are kept in the git tracker instead

Reply:
@saurabhintoml still to remove?